src/concrete/ml/quantization/__init__.py

```diff
@@ -1,6 +1,11 @@
 """Modules for quantization."""
 from .base_quantized_op import QuantizedOp
-from .post_training import PostTrainingAffineQuantization, PostTrainingQATImporter, get_n_bits_dict
+from .post_training import (
+    PostTrainingAffineQuantization,
+    PostTrainingQATImporter,
+    get_n_bits_dict,
+    get_n_bits_dict_trees,
```

Comment:
I added a new get_n_bits for trees, to avoid changing the behavior of the existing method that uses op_weights

Reply:
now that I think of it, I definitively get the relevancy behind having an `op_leaves` instead of `op_weights`. However, we are therefore introducing a new name and making this `n_bits` parameter more complex to users (which are already not supposed to be familiar with it) , so maybe we should discuss about that. 

Personally, I think I wouldn't be against keeping the `op_weights` even for trees for that particular reason

Reply:
by the way I would start to make this fund private (will do the same for `get_n_bits_dict` later : https://github.com/zama-ai/concrete-ml-internal/issues/3995)